codePuls.Application/Services/CodeRepositoryService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using codePuls.Application.DTOs.RequestDTOs;
+using codePuls.Application.DTOs.ResponseDTOs;
+using codePuls.Application.Interfaces;
+using codePuls.Domain.Entities;
+using CodePuls.Domain.Interfaces.Repositories;
+using Mapster;
+using Microsoft.Extensions.Logging;
+
+namespace codePuls.Application.Services
+{
+    public class CodeRepositoryService(ICodeReositoryReository codeRepositoryService, ILogger<ProjectService> logger) : ICodeRepositoryService
+    {
+        private readonly ICodeReositoryReository _codeRepositoryrepository = codeRepositoryService;
```

Comment:
May be use `ICodeRepoRepository` ? Just a suggestion. Having 2 repository in the word makes little bit weired.
```suggestion
        private readonly ICodeRepositoryRepository _codeRepositoryRepository = codeRepositoryService;
```